src/Microsoft.Xaml.Behaviors/Core/LaunchUriOrFileAction.cs

```diff
@@ -34,8 +34,13 @@ public string Path
         protected override void Invoke(object parameter)
         {
             if (this.AssociatedObject != null && !string.IsNullOrEmpty(this.Path))
-            {
-                Process.Start(this.Path);
+            {
+                var processStartInfo = new ProcessStartInfo(this.Path)
+                {
+                    CreateNoWindow = true,

```

Comment:
Why this change? The default is false on both WPF and .NET Core. I think it's safer to not set this, so this doesn't change behavior, and it seems to continue to work without setting it.

Reply:
I guess I overlooked the default on .NET Core, it's not needed. I removed it!